src/Panel.vue

```diff
@@ -28,12 +28,12 @@
                                       @click.native.stop.prevent="expand()"
                                       @is-open-event="retrieveOnOpen" :is-light-bg="isLightBg"></panel-switch>
                         <button type="button" :class="['close-button', 'btn', isLightBg ? 'btn-outline-secondary' : 'btn-outline-light']"
-                                v-show="!isSeamless ? (!noCloseBool) : onHeaderHover"
+                                v-show="isSeamless ? onHeaderHover : (!noCloseBool)"
                                 @click.stop="close()">
                             <span class="glyphicon glyphicon-remove" aria-hidden="true"></span>
                         </button>
                         <button type="button" :class="['popup-button', 'btn', isLightBg ? 'btn-outline-secondary' : 'btn-outline-light']"
-                                v-show="this.popupUrl !== null"
+                                v-show="((this.popupUrl !== null) && (isSeamless ? onHeaderHover : true))"
```

Comment:
`(isSeamless ? onHeaderHover : true))` is a form of `p -> q`.

Therefore, this can be simplified to:
```
(this.popupUrl !== null) && (!isSeamless || onHeaderHover)
```

Reply:
Nice.